TidepoolOnboardingKitUI/TidepoolOnboardingUI.swift

```diff
@@ -29,11 +29,18 @@ public final class TidepoolOnboardingUI: ObservableObject, OnboardingUI {
 
     var therapySettings: TherapySettings? {
         didSet {
-            guard therapySettings != oldValue, let therapySettings = therapySettings else { return }
+            guard let therapySettings = therapySettings else { return }
             notifyHasNewTherapySettings(therapySettings)
         }
     }
 
+    var dosingEnabled: Bool? {
+        didSet {
+            guard let dosingEnabled = dosingEnabled else { return }
+            notifyHasNewDosingEnabled(dosingEnabled)
```

Comment:
[nit] what should happen in the `nil` case, especially if it becomes `nil` from `true`? Should this have a default value?

Reply:
The default of `nil` means that it hasn't been set by the onboarding UI yet and is expected/correct. However, you are right that it should never be **set** to `nil` so I suppose I could use a precondition here instead of a guard. I'll do that.